src/App.jsx

```diff
@@ -1,14 +1,21 @@
+import { Route, Routes } from 'react-router-dom'
 import './App.css'
 import Home from './Components/Home'
+import SingleArticle from './Components/SingleArticle'
+import NavBar from './Components/NavBar'
 
 function App() {
   
   return (
           <>
           <header>
+            <NavBar/>
           </header>
           <main>
-            <Home/>
+            <Routes>
+              <Route path='/' element={<Home/>}/>
+              <Route path='/article/:article_id' element={<SingleArticle/>}/>
```

Comment:
For this path '/articles/:article_id' instead of '/article/:article_id' would be more appropriate.